npm/angular/src/mount.ts

```diff
@@ -177,9 +177,14 @@ export class CypressTestComponentRenderer extends TestComponentRenderer {
     this.removeAllRootElements()
 
     const rootElement = getContainerEl()
+    const parentElement = rootElement.parentElement
 
     rootElement.setAttribute('id', rootElId)
-    document.body.appendChild(rootElement)
+    if (parentElement && parentElement?.tagName !== 'HTML') {
+      parentElement.appendChild(rootElement)
+    } else {
+      document.body.appendChild(rootElement)
+    }
```

Comment:
```suggestion
```

The true fix here is the removal of `document.body.appendChild(rootElement)`, we don't need this extra logic since `rootElement` is already guaranteed to be in the DOM (`getContainerEl()` uses querySelector, so if it wasn't in the DOM it will throw)

Reply:
Oh, I didn't see the mount.cy.ts. You are right, it is a much better idea to inject the different HTML during the test. 

I will update my branch based on your suggestion